src/hotspot/cpu/x86/x86.ad

```diff
@@ -1468,6 +1468,11 @@ const bool Matcher::match_rule_supported(int opcode) {
         return false;
       }
       break;
+    case Op_PopulateIndex:
+      if (!is_LP64) {
+        return false;
+      }
+      // fallthrough
```

Comment:
Please, don't do `fallthrough` - `RoundVF` is not related to `PopulateIndex`. Why not `if (!is_LP64 || UseAVX < 2)`?
Are there limitations in 32 bits or you don't want spend time on not major platform (which is also understandable)?

Reply:
It is not a major platform so I didn't spend time on it.